.gitignore

```diff
@@ -15,3 +15,4 @@ phpcs.xml
 yarn.lock
 docker-compose.override.yml
 cypress.env.json
+*.swp
```

Comment:
Can you explain what this file is? Typically files which aren't specific to the project or otherwise common to encounter should be added to your own global `gitignore`:

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Ah, no problem. That was to ignore swap files that Vim generates when editing files. They're on by default in Vim, so I figured it might be nice to ignore those for other Vim users. I can still take it out in the `.gitignore` if you'd like, though.